src/python/pants/backend/python/tasks/python_binary_create.py

```diff
@@ -38,6 +38,16 @@ def register_options(cls, register):
                   "created and the command line used to create it. This information may be helpful to you, but means "
                   "that the generated PEX will not be reproducible; that is, future runs of `./pants binary` will not "
                   "create the same byte-for-byte identical .pex files.")
+    # TODO: make an analogy to cls.register_jvm_tool that can be overridden for python subsystems
+    # by a python_requirement_library() target, not just via pants.ini!
+    register('--ptex-injected-pex-version', advanced=True, default='1.6.12', fingerprint=True,
```

Comment:
Again, please accept my apologies if the kill is in-flight, but this should go. Just use:
```python
from pex.version import __version__ as pex_version
f'pex=={pex_version}'
```